js/apps/account-ui/src/root/PageNav.tsx

```diff
@@ -134,9 +132,7 @@ export const NavLink = ({
   isActive,
   children,
 }: PropsWithChildren<NavLinkProps>) => {
-  const { realm } = useParams();
-
-  const menuItemPath = `${getRootPath(realm)}/${to}`;
```

Comment:
I think this is the only place `getRootPath()` was used in the actual code that gets bundled. So perhaps we should move this utility out of `src`? Or perhaps even better, we could re-write `getRootPath()` to return `new URL(environment.baseUrl).pathname`.

Reply:
TL;DR: This particular issue is a release blocker since it renders `--http-relative-path` option unusable. The `getRootPath` removal which was present in the first version of this PR turned out not really straightforward failing the tests. So I'd prefer this refactor be done separately.

Longer version:
The `getRootPath` is parameterized with realm name, and this parameterization is used in linked accounts tests. Hence it cannot be easily exchanged with the `baseUrl` which contains already realm name hardcoded. Also `getRootPath` turned out to be called from several other places and the account console tests failed.

Reply:
Introduced #27716 so that this refactor is not forgotten.